doc/compile.rst

```diff
@@ -186,9 +186,9 @@ Build and install libupnp with the ``--enable-ipv6`` and ``--enable-reuseaddr``
 
 ::
 
-  wget "https://downloads.sourceforge.net/project/pupnp/pupnp/libupnp-1.12.1/libupnp-1.12.1.tar.bz2?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fpupnp%2Ffiles%2Flatest%2Fdownload&ts=1588248015" -O libupnp-1.12.1.tar.bz2
-  tar -xf libupnp-1.12.1.tar.bz2
-  cd libupnp-1.12.1
+  wget "https://github.com/pupnp/pupnp/releases/download/release-1.14.10/libupnp-1.14.10.tar.bz2" -O libupnp-1.14.10.tar.bz2
```

Comment:
Nice spot!

Reply:
Up until now, I was actually following that script and had 1.12.1 installed, which required me to make local code changes so I can compile. I was too lazy to track down this problem :)

Reply:
Just run `scripts/install-*.sh` and feel happy